server/views/resources/shop.jade

```diff
@@ -12,20 +12,60 @@ block content
                 <form action="//www.paypal.com/cgi-bin/webscr" onSubmit="ga('send', 'event', 'SHOP', 'SHOP-STICKER', 'Sticker PayPal button clicked');" method="post" target="_blank">
                 <input type="hidden" name="cmd" value="_s-xclick">
                 <input type="hidden" name="hosted_button_id" value="R9AGYKJUCU37N">
-                <input type="image" src="//www.paypalobjects.com/en_US/i/btn/btn_buynowCC_LG.gif" border="0" name="submit" alt="PayPal - The safer, easier way to pay online!">
+                <input type="submit" class="btn btn-cta signup-btn btn-block" name="submit" value='Buy now with PayPal or a Credit Card'>
                 <img alt="" border="0" src="//www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
                 </form>
     hr
     .row
-        .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3.text-center
+        .col-xs-12.col-sm-10.col-sm-offset-1.col-md-8.col-md-offset-2.text-center
             img.img-responsive.img-center(src='//i.imgur.com/BTtb8P0g.jpg')
             h3 Get a $10 credit your own Digital Ocean server.
```

Comment:
~~Get a $10 credit your own Digital Ocean server.~~ Get $10 credit for your own Digital Ocean server.